File: compat/src/main/java/androidx/preference/MultiSelectListPreference.java
package androidx.preference;

import android.content.Context;

import java.util.Set;

public class MultiSelectListPreference extends DialogPreference {
	public CharSequence[] entries, entryValues;
	public Set<String> values;
	
	public MultiSelectListPreference(Context context) {}
	
	public void setEntries(CharSequence[] entries) {
		this.entries = entries;
	}
	
	public CharSequence[] getEntries() {
		return entries;
	}
	
	public void setEntryValues(CharSequence[] entryValues) {
		this.entryValues = entryValues;
	}
	
	public CharSequence[] getEntryValues() {
		return entryValues;
	}
	
	public void setValues(Set<String> values) {
		this.values = values;
	}
	
	public Set<String> getValues() {
		return values;
	}
	
	public int findIndexOfValue(String values) {
		throw new UnsupportedOperationException();
	} 
}

File: extension/loaders/android-compat/src/desktopMain/kotlin/androidx/preference/MultiSelectListPreference.kt
package androidx.preference

import android.content.Context
import com.mrboomdev.awery.core.utils.PlatformSdk

@PlatformSdk
class MultiSelectListPreference(
	context: Context
): DialogPreference(context) {
	private var entries: Array<CharSequence>? = null
	private var entryValues: Array<CharSequence>? = null

	@PlatformSdk
	fun setEntries(entries: Array<CharSequence>?) {
		this.entries = entries
	}

	@PlatformSdk
	fun getEntries(): Array<CharSequence>? {
		return entries
	}

	@PlatformSdk
	fun setEntryValues(entryValues: Array<CharSequence>?) {
		this.entryValues = entryValues
	}

	@PlatformSdk
	fun getEntryValues(): Array<CharSequence>? {
		return entryValues
	}

	@PlatformSdk
	fun setValues(values: Set<String>?) {
		throw RuntimeException("Stub!")
	}

	@PlatformSdk
	fun getValues(): Set<String>? {
		throw RuntimeException("Stub!")
	}

	@PlatformSdk
	fun findIndexOfValue(value: String?): Int {
		throw RuntimeException("Stub!")
	}
}